.github/workflows/clamav.yml

```diff
@@ -0,0 +1,46 @@
+name: Clamav - Install/Upgrade/Remove
+run-name: Clamav -  ${{ inputs.environment }} Clamav in ${{ inputs.environment }} using ${{ inputs.gitRef }}
```

Comment:
Minor but I believe that we should call it ClamAV everywhere where it is not a variable name.